samza-core/src/main/java/org/apache/samza/coordinator/metadatastore/CoordinatorStreamStore.java

```diff
@@ -149,6 +136,13 @@ private void putWithoutFlush(String namespacedKey, byte[] value) {
     systemProducer.send(SOURCE, envelope);
   }
 
+  @Override
+  public void putAll(Map<String, byte[]> entries) {
```

Comment:
do we need this impl of `putAll` because `MetadataStore.putAll` default impl in the interface `MetadatStore` is the same?

Reply:
Good catch @lakshmi-manasa-g . This function is duplicated with the one in `MetadataStore`, I will delete it.